### PR TITLE
Add ability to post comments

### PR DESCRIPTION
**Pull request by stevenpetryk**
Based on #31, so that'll need to be merged first before this diff makes sense.

### CODE REVIEW

**Review by GregKWhite — APPROVED** [review]
Looks like react to me

File: src/actions/comment.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {
+  fetch,
+  createResource
+} from './index'
+
+export function fetchCommentsForStream (streamId) {
+  return fetch({
+    url: `/streams/${streamId}/comments`,
+    types: ['FETCH_COMMENTS_REQUEST', 'FETCH_COMMENTS_SUCCESS', 'FETCH_COMMENTS_REQUEST'],
```
Comment (maxcell) [inline src/actions/comment.js]: What is the difference between the two `'FETCH_COMMENTS_REQUEST'` ?

Comment (maxcell) [reply src/actions/comment.js]: Looks like one should be `'FETCH_COMMENTS_FAILURE'`

**Review by maxcell — CHANGES_REQUESTED** [review]

File: src/actions/comment.js (hunk 1 above)
Comment (stevenpetryk) [reply src/actions/comment.js]: Good catch!

**Review by maxcell — APPROVED** [review]

**Pull request merged**